modules/pipeline/providers/clusterinfo/interface.go

```diff
@@ -91,6 +95,15 @@ func (p *provider) BatchUpdateAndDispatchRefresh() error {
 }
 
 func (p *provider) batchUpdateClusterInfo() error {
+	// first the edge and center both get their current cluster info from k8s configmap
+	currentClusterInfo, err := p.GetCurrentClusterInfoFromK8sConfigMap()
+	if err != nil {
+		p.Log.Warnf("failed to get current cluster info(continue list clusters), err: %v", err)
+	} else {
+		p.cache.UpdateClusterInfo(currentClusterInfo)
+	}
+	// then center continue get all clusters from cluster-manager
+	// and will cover the current cluster info, it's acceptable
 	clusterInfos, err := p.bdl.ListClusters("", 0)
```

Comment:
I think it will always be failed at edge side.

Reply:
adjusted, if deploy at edge side, return directly